boot/boot_serial/src/boot_serial.c

```diff
@@ -95,12 +95,30 @@ BOOT_LOG_MODULE_DECLARE(mcuboot);
 #else
 #define BOOT_SERIAL_HASH_SIZE_MAX 0
 #endif
+#ifdef MCUBOOT_SERIAL_IMG_GRP_SLOT_INFO
+#define BOOT_SERIAL_SLOT_INFO_SIZE_MAX 164
+#else
+#define BOOT_SERIAL_SLOT_INFO_SIZE_MAX 0
+#endif
+
+#if (128 + BOOT_SERIAL_IMAGE_STATE_SIZE_MAX + BOOT_SERIAL_HASH_SIZE_MAX) > \
```

Comment:
I think this requires description, specifically that there is the 128 literal without explanation.

Reply:
The 128 is actually not from me, it is from the code before, why is 128 there? No clue